### PR TITLE
[Markdown]: Fix markdown stub code to pass newly added tests

### PR DESCRIPTION
**Pull request by Jasstkn**
Hi! I found out that after changes in https://github.com/exercism/python/pull/2753 3 tests didn't pass despite of the task statement:
> Even though this code is confusingly written and hard to follow, somehow it works and all the tests are passing!

This PR fixes this and adds required conditions.

Signed-off-by: Jasstkn <jasssstkn@yahoo.com>

### CODE REVIEW

**Comment by J08K** [comment]
Hey, nice figuring this out! Thanks so much.
image
This indeed does work. Thank you so much for fixing this.

**Review by J08K — APPROVED** [review]

**Comment by Jasstkn** [comment]
> Hey, nice figuring this out! Thanks so much. image This indeed does work. Thank you so much for fixing this.

Hello! I'm very glad to be able to help :) Thanks for reviewing this PR!
